docs/learn-scams.md

```diff
@@ -0,0 +1,38 @@
+---
+id: learn-scams
+title: How to Protect Yourself from Scams
+sidebar_label: How to Protect Yourself from Scams
+---
+
+Scams have unfortunately made an impact on our community. It's important to stay alert and protect
+yourself and your non-refundable crypto assets from scammers. If you ever feel you are being
+scammed, please email [our support team](mailto:support@web3.foundation).
+
```

Comment:
Essential rules:
1. Never, ever, ever share your seed phrase or account password
1. Do not trust anyone online.  It is trivial for them to lie, change identities.
1. If you are scammed, there is likely nothing that can be done to recover your funds.  If a scammer gets a hold of your seed phrase, they can transfer all of your funds to their account in seconds.  It is better to be safe than to risk all of DOT and KSM.
1. If it sounds too good to be true, it probably is.  People, especially celebrities, do not give away crypto for free.  Even if they wanted to, they could just ask for your address as opposed to having you send them tokens.

Reply:
5. Scams are absolutely rife in this space.  It is easy and cheap to set a scam up, and hard to shut one down.  Therefore, the onus is on the user to be as diligent as possible in avoiding them.